ansible_collections/arista/avd/molecule/digital_twin/inventory/hosts.yml

```
@@ -14,15 +14,19 @@ all:
                 use_default_interfaces_of_digital_twin_platform: true
         CLEAN_UNIT_TESTS: # Single Devices testing one specific case and only using hostvars
           hosts:
-            # No eAPI. Automatically enabling eAPI over TLS in default VRF.
             digital-twin-enforce-eapi-1:
               type: l2leaf
-            # Default eAPI settings. Automatically adding default VRF to the eAPI scope.
             digital-twin-enforce-eapi-2:
               type: l2leaf
-            # eAPI enabled in default VRF. IPv4 ACL applied. Automatically clearing IPv4 ACL.
             digital-twin-enforce-eapi-3:
               type: l2leaf
-            # eAPI not enabled in default VRF. Automatically enabling it.
             digital-twin-enforce-eapi-4:
               type: l2leaf
+            digital-twin-adjust-oob-mgmt-1:
+              type: l2leaf
+            digital-twin-adjust-oob-mgmt-2:
+              type: l2leaf
+            digital-twin-adjust-oob-mgmt-3:
+              type: l2leaf
+            digital-twin-adjust-oob-mgmt-4:
+              type: l2leaf
```

These are already in the host vars where they fit better.
```suggestion
            digital-twin-adjust-oob-mgmt-1:
            digital-twin-adjust-oob-mgmt-2:
            digital-twin-adjust-oob-mgmt-3:
            digital-twin-adjust-oob-mgmt-4:
```

---

indeed, removed for both new and old hosts. Fixed